src/libs/SidebarUtils.ts

```diff
@@ -292,8 +302,10 @@ function getOptionData(
         isAllowedToComment: true,
         isDeletedParentAction: false,
     };
+
     const participantPersonalDetailList: PersonalDetails[] = Object.values(OptionsListUtils.getPersonalDetailsForAccountIDs(report.participantAccountIDs ?? [], personalDetails));
     const personalDetail = participantPersonalDetailList[0] ?? {};
+    const hasErrors = Object.keys(result.allReportErrors ?? {}).length !== 0;
```

Comment:
Defining `hasErrors` before `result.allReportErrors` on line 320 resulted in the [following regression](https://github.com/Expensify/App/issues/35061), which should've ideally been caught before deploy by having more robust offline testing!